.github/workflows/bsim-tests-bluetooth.yaml

```diff
@@ -3,12 +3,12 @@ name: BabbleSim Tests
 on:
```

Comment:
Then, it would be a bit nicer to rename this workflow to BabbleSim BT Tests

Reply:
fixed